### DISCUSSION TITLE
PhysX 5.4.0

### DISCUSSION THREAD

**Discussion by ParkSeWon1997** [NVIDIA-Omniverse/PhysX#303] [Q&A]
Hi!
I'm currently developing using Directx11
In my project, I am using PhysX 5.4.0 to give my objects physical effects.
But I 
I'm loading my .fbx and drawing the model in my game world.
But I want to destroy this model.
It's like creating an effect where a tank destroys a wall when it fires a cannon.
But when I looked it up, it said that I could use the Blast API from Nividia.
How do I port the Blast API to my project?
I need a detailed explanation
Or is there a document I should refer to?

**Comment by vreutskyy** [comment]
Hi @Jusin142. Did you try to read [Blast SDK Documentation](https://nvidia-omniverse.github.io/PhysX/blast/index.html)?

**Reply by ParkSeWon1997** [reply]
yes I read the document
My limitation is as far as linking the Blast lib into my project.
Only the build was successful.
I think the document contains only unkind content.

I just want to initialize Blast in some way, split my Model, and get the results of the split vertices. 
Oh, and linking it with PhysX RigidActor as well.
